src/pages/Calculator.js

```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+import Calculator from '../components/Calculator';
+import Qoute from '../components/qoute';
+import './style.css';
+
+const CalculatorPage = () => (
+  <div className="page_container">
+    <div className="qoute_wrapper">
+      <Qoute />
+    </div>
+    <div className="digit_wrapper">
+      <Calculator />
+    </div>
+  </div>
+);
```

Comment:
- [ ] Kindly design the calculator page to have the same layout as the wireframe provided. The Calculator page doesn't have the quote being fetched. Rather, it has the _**"Let's Do Some Math!"**_ text. Kindly follow the design of the wireframe

![image](https://user-images.githubusercontent.com/77106962/228366775-e2d40c37-9174-46ba-86ad-05c94164f241.png)